simapp/app.go

```diff
@@ -569,6 +614,7 @@ func NewSimApp(
 		mockModule,
 
 		// IBC light clients
+		wasm.NewAppModule(app.WasmClientKeeper), // TODO(damian): see if we want to pass the lightclient module here, keeper is used in AppModule.RegisterServices etc
```

Comment:
copied this over, is this todo still relevant? I'm not sure I see how the light client module could be passed?